src/hdx_cli_toolkit/ckan_utilities.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+import json
+import urllib3
+
+from collections import Counter
+
+import ckanapi
+
+from hdx_cli_toolkit.hdx_utilities import get_hdx_url_and_key, configure_hdx_connection
+from hdx_cli_toolkit.utilities import query_dict
+
+
+def fetch_data_from_ckan_package_search(
+    query_url: str, query: dict, hdx_api_key: str, fetch_all: bool = False
+) -> dict:
+    headers = {
+        "Authorization": hdx_api_key,
+        "Content-Type": "application/json",
+    }
+
+    start = 0
+    if "start" not in query.keys():
+        query["start"] = start
+    if "rows" not in query.keys():
+        query["rows"] = 10

```

Comment:
Where does this 10 come from?

Reply:
`Rows` is the `ckanapi` equivalent of `limit` and 10 is the default value I picked / default if queried in the browser. I think I'll put it in a constant and set it to 100.